Makefile

```diff
@@ -134,28 +136,87 @@ clean-linux:
 	@echo "> Building 'nanoserver-1809' stack..."
 	bash stacks/build-stack.sh stacks/nanoserver-1809
 
-deploy-windows: deploy-windows-stacks
+build-windows-builders: build-builder-nanoserver-1809
+
+build-builder-nanoserver-1809: build-sample-root # build-windows-packages: not yet supported
+	@echo "> Building 'nanoserver-1809' builder..."
+	$(PACK_CMD) create-builder cnbs/sample-builder:nanoserver-1809 --config $(SAMPLES_ROOT)/builders/nanoserver-1809/builder.toml $(PACK_FLAGS)
+
+build-windows-buildpacks: build-buildpacks-nanoserver-1809
+
+build-buildpacks-nanoserver-1809:
+	@echo "> Creating 'hello-world-windows' app using 'nanoserver-1809' builder..."
+	$(PACK_CMD) build sample-hello-world-windows-app:nanoserver-1809 --builder cnbs/sample-builder:nanoserver-1809 --buildpack $(SAMPLES_ROOT)/buildpacks/hello-world-windows $(PACK_FLAGS) $(PACK_BUILD_FLAGS)
+
+build-windows-apps: build-sample-root
+	@echo "> Creating 'batch-script' app using 'nanoserver-1809' builder..."
+	$(PACK_CMD) build sample-batch-script-app:nanoserver-1809 --builder cnbs/sample-builder:nanoserver-1809 --path apps/batch-script $(PACK_FLAGS) $(PACK_BUILD_FLAGS)
+
+deploy-windows: deploy-windows-stacks deploy-windows-builders # deploy-windows-packages: not yet supported
 
 deploy-windows-stacks:
 	@echo "> Deploying 'nanoserver-1809' stack..."
 	docker push cnbs/sample-stack-base:nanoserver-1809
 	docker push cnbs/sample-stack-run:nanoserver-1809
 	docker push cnbs/sample-stack-build:nanoserver-1809
 
+deploy-windows-builders:
+	@echo "> Deploying 'alpine' builder..."
+	docker push cnbs/sample-builder:nanoserver-1809
+
 clean-windows:
 	@echo "> Removing 'nanoserver-1809' stack..."
-	docker rmi cnbs/sample-stack-base:nanoserver-1809
-	docker rmi cnbs/sample-stack-run:nanoserver-1809
-	docker rmi cnbs/sample-stack-build:nanoserver-1809
+	docker rmi cnbs/sample-stack-base:nanoserver-1809 || true
+	docker rmi cnbs/sample-stack-run:nanoserver-1809 || true
+	docker rmi cnbs/sample-stack-build:nanoserver-1809 || true
+
+	@echo "> Removing builders..."
+	docker rmi cnbs/sample-builder:nanoserver-1809 || true
+
+	@echo "> Removing 'nanoserver-1809' apps..."
+	docker rmi sample-hello-world-windows-app:nanoserver-1809 || true
+	docker rmi sample-batch-script-app:nanoserver-1809 || true
+
+	@echo "> Removing '.tmp'"
+	rm -rf .tmp
+
+
+####################
+## Windows pack for any daemon OS
+####################
+
+# pack on Windows does not support directory-based buildpacks
+# workaround by pivoting samples-root to tmp path with tgz-buildpacks of the same name
+ifeq ($(OS),Windows_NT)
+SAMPLES_ROOT:=$(shell mkdir -p .tmp && mktemp --directory -p . .tmp/samples-XXX)
+build-sample-root:
+	@mkdir -p $(SAMPLES_ROOT)/buildpacks/
+
+	@for bp_dir in ./buildpacks/*/; do \
+		tar -czf $(SAMPLES_ROOT)/buildpacks/$$(basename $$bp_dir) -C $$bp_dir . ; \
+	done
+
+	@cp -r builders packages $(SAMPLES_ROOT)/
+else
+# No-op for posix pack
+SAMPLES_ROOT:=.
+build-sample-root:
+endif
```

Comment:
This makes `build-linux-` targets work for LCOW for anyone using `bash.exe`, and also let's `build-windows-` targets work for WCOW (if they are run from a Mac using DOCKER_HOST, this branch is not used).

This can all be removed once directory buildpacks are supported for pack on Windows.